crates/burn-tensor/src/tensor/api/numeric.rs

```diff
@@ -736,6 +738,49 @@ where
             indices.select(dim, k_indices),
         )
     }
+
+    /// Pad the tensor with the given value on the last two dimensions.
+    ///
+    /// # Arguments
+    ///
+    /// * `pads` - A tuple of four integers representing the padding on the left, right, top, and bottom.
+    /// * `value` - The value to pad the tensor with.
+    ///
+    /// # Returns
+    ///
+    /// A new tensor with the given padding.
+    pub fn pad(self, pads: (usize, usize, usize, usize), value: K::Elem) -> Tensor<B, D, K> {
+        let (left, right, top, bottom) = pads;
```

Comment:
Small nitpick: can we rename `pads` to `padding` (or even `pad` could work, the plural just sounds weird)?

Reply:
Renamed to padding